examples/advanced/quantum_volume.py

```diff
@@ -191,4 +304,14 @@ def parse_arguments(args):
 
 
 if __name__ == '__main__':
-    main(**parse_arguments(sys.argv[1:]))
+    device = cirq.google.Bristlecone
```

Comment:
put this code in a function. I'd suggest: this code goes in `main()` and you rename `main` to something like `calculate_quantum_volume` (or something meaningful)

reason: you're actually declaring global variables here. PyCharm tells me that your usage of the variable `optimize` in other functions shadows this outer definition which is probably fine in practice but could be bad

Reply:
Ah, good to know, thanks! Didn't realize it, but in retrospect it's pretty clear that this is in the global scope.